dfx.json

```diff
@@ -6,6 +6,15 @@
 			"build": "scripts/build.signer.sh",
 			"candid": "src/signer/canister/signer.did",
 			"wasm": "out/signer.wasm.gz",
+			"metadata": [
+				{
+					"name": "candid:service"
+				},
+				{
+					"name": "git_commit_id",
+					"path": "target/commit"
+				}
+			],
```

Comment:
We currently add the metadata ourselves, but it is polite to announce the metadata in the standard place, even if dfx doesn't add it.

Reply:
This is not standard dfx.json?

Reply:
Yes but... in the case of a custom build it does nothing.  If we get more {pre,post}-{build,deploy} hooks then we could use native dfx build commands rather than custom builds and then dfx would populate these fields for us.

Reply:
Resolving so that I can merge but happy to discuss further if wished.